client/shared/src/util/useObservable.ts

```diff
@@ -22,12 +22,12 @@ export function useError(): (error: any) => void {
  * @param observable The Observable to subscribe to. If this is the return value of a function, you should use `useMemo()` to make sure it is not resubscribed on every render.
  * @throws If the Observable pipeline errors.
  */
-export function useObservable<T>(observable: Observable<T>): T | undefined {
+export function useObservable<T>(observable: Observable<T> | BehaviorSubject<T>): T | undefined {
+    const [currentValue, setCurrentValue] = useState<T | undefined>((observable as BehaviorSubject<T>).value)
```

Comment:
Should this be

```suggestion
    const [currentValue, setCurrentValue] = useState<T | undefined>(() => {
        if (observable instanceof BehaviorSubject) {
            return observable.value
        }
        return undefined
    })
```
? This seems unsafe.

Reply:
Same in the `useEffect` below.

Reply:
That was the first thing I've tested, but this check didn't work because `LocalStorageSubject` doesn't extend `BehaviorSubject`. So an additional type-guard was needed. But I will go with the approach that @felixfbecker suggested: no changes in the `useObservable` hook.